bpf/kmesh/workload/xdp.c

```diff
@@ -122,6 +186,15 @@ int xdp_shutdown(struct xdp_md *ctx)
 
     // never failed
     parser_tuple(&info, &tuple_info);
+#ifdef AUTH_BY_XDP
```

Comment:
From here i can see if AUTH_BY_XDP defined, the userspace result is not checked, but in sockops, it is still reporting to userspace. Can we also respect the macro there